src/components/calcite-date-picker/calcite-date-picker.e2e.ts

```diff
@@ -121,7 +121,7 @@ describe("calcite-date-picker", () => {
   });
 
   describe("when the locale is set to Slovak calendar", () => {
-    it("should start the week on Monday", async () => {
+    it.skip("should start the week on Monday", async () => {
```

Comment:
I submitted https://github.com/Esri/calcite-components/pull/2899 to keep the test-skip separate. The reason is if we had to revert or do something with the squashed commit for the fix, it would affect an unrelated test.